definitions/npm/humanize-number_v0.0.x/flow_v0.53.x-/humanize-number_v0.0.x.js

```diff
@@ -0,0 +1,11 @@
+type $npm$HumanizeNumber$Options = {
+  delimiter: string,
+  separator: string
```

Comment:
You'll need to make the presence of these unrequired, to successfully run `humanize(100, { delimiter: " " });`:

```
type $npm$HumanizeNumber$Options = {
  delimiter?: string,
  separator?: string
};
```